hack/swift/Makefile

```diff
@@ -92,6 +92,20 @@ overlay-byocni-up: rg-up overlay-net-up ## Brings up an Overlay BYO CNI cluster
 		--yes
 	@$(MAKE) set-kubeconf
 
+overlay-cilium-up: rg-up overlay-net-up ## Brings up an Overlay AzCNI cluster
+	$(AZCLI) aks create -n $(CLUSTER) -g $(GROUP) -l $(REGION) \
+		--node-count 2 \
+		--node-vm-size $(VM_SIZE) \
+		--load-balancer-sku basic \
```

Comment:
Info Question: Are we tied to using only basic SKUs for Cilium? SLB recommendations was to move to Stnadard SKUs across the board so that they could deprecate Basic SKUs

Reply:
No, these are not limitations. I have just tended towards the cheapest config possible since this is targeted for quick cluster creation